muhireheir/mybrand-1
# Ch server setup

> #### What does this PR do?
> * create nodejs server
> * connect to mongodb

> #### Description of Task to be completed
> * to create a server for my project
> #### How should this be manually tested?
>  * clone the repository
> * change to `ch-serverSetup` branch
> * open  terminal  
> * run  the  following   codes `npm install` and wait until packages are finished to install
> * run  the  following   codes `npm start`
> * open postman or browser
> * in  url enter the following adress `localhost:2000`
> * send request with any http method
> * see  the image below  to have see the expected result
> * Done.
> #### Any background context you want to provide?
> * 
> #### What are the relevant Trello story?
>  * [https://trello.com/c/qSoD6qVu](https://trello.com/c/qSoD6qVu)
> #### Screenshots (if appropriate)

![folder](https://user-images.githubusercontent.com/26524106/94013895-0eeefe00-fdab-11ea-9804-d33d7c216315.png)
